sysrib/server.go

```diff
@@ -622,6 +578,66 @@ func (s *Server) ResolveAndProgramDiff() error {
 	return nil
 }
 
+func (s *Server) resolveAndProgramDiffAux(niName string, ni *NIRIB, address string, newResolvedRoutes map[RouteKey]*Route) {
```

Comment:
is there a more descriptive name for this?

Reply:
It's a helper function and what it does is strange as a standalone function.

I've added comments to clarify its behaviour.